website/src/blogs/data-migration-within-kubernetes.md

```diff
@@ -0,0 +1,152 @@
+This blog is for OpenEBS users, specifically **cStor CSI** users looking to **migrate data within Kubernetes** from one node to another without any downtime of the application.
+
+![Data Migration from Kubernetes from Node2 to Node3](/images/blog/2020/07/Data-migration-diagram.png)
+
+### Create cStor Pools(CSPC):
+
+Create cStor pools by following the steps mentioned [here](https://github.com/openebs/cstor-operators/blob/master/docs/quick.md). Once the pools are created, wait till all the cStor pools marked as Healthy. Check the cStor pools status by executing `kubectl get cspc -n openebs` command(cspc - cStorPoolCluster)
+
+    NAME            HEALTHYINSTANCES   PROVISIONEDINSTANCES   DESIREDINSTANCES   AGE
+    cspc-stripe-pool   3                  3                       3              4m13s
+
+Following command `kubectl get cspi -n openebs`(cStorPoolInstances) will be helpful to know more information about the pool.
+
+    NAME                   HOSTNAME   ALLOCATED FREE CAPACITY READONLY  TYPE   STATUS  AGE
+    cspc-stripe-pool-6qkw  e2e1-node1   230k    9630M   10G   false   stripe   ONLINE  21m
+    cspc-stripe-pool-pn9p  e2e1-node2   230k    9630M   10G   false   stripe   ONLINE  12s
+    cspc-stripe-pool-psz5  e2e1-node3   230k    9630M   10G   false   stripe   ONLINE  21m
+
+### Create CSI Volume:
+
+Create CSI volumes on cStor pools created above by following the steps mentioned [here](https://github.com/openebs/cstor-csi#provision-a-cstor-volume-using-openebs-cstor-csi-driver). As part of volume provisioning, a resource called `CStorVolumeConfig` will be created. Once the volume is provisioned successfully, then CVC(cStorVolumeConfig) status will update to Bound, which means all the CStorVolume resources are successfully created. Following is the command which will help to get CVC status `kubectl get cvc -n openebs`.
+
+    NAME                                       STATUS     AGE
+    pvc-d1b26676-5035-4e5b-b564-68869b023306   Bound      5m56s
+    
+
+### **Inspect CVC**:
+
+Interfere CVC to know on which node data exists. When we do `kubectl get cvc <PV_NAME> -n <openebs_namespace> -o yaml`.
+
+    apiVersion: cstor.openebs.io/v1
+    kind: CStorVolumeConfig
+    name: pvc-d1b26676-5035-4e5b-b564-68869b023306
+    …
+    …
+    spec:
+      capacity:
+        storage: 5Gi
+    ...
+    ...
+      policy:
+        replicaPoolInfo:
+        - poolName: cspc-stripe-pool-6qkw
+        - poolName: cspc-stripe-pool-pn9p
+    status:
+      phase: Bound
+      poolInfo:
+      - cspc-stripe-pool-6qkw
+      - cspc-stripe-pool-pn9p
+
+From the above info CStorVolumeReplicas(CVR) are scheduled on cStor pools **cspc-stripe-pool-6qkw** and **cspc-stripe-pool-pn9p** from the **status.poolInfo** since above pools are on **e2e1-node1** and **e2e1-node2** (able to find node info from cspi output)sodata also will be available on the same node. Info under spec i.e **spec.policy.replicaPoolInfo** will convey where to schedule cStorVolumeReplicas.
```

Comment:
```suggestion
From the above info CStorVolumeReplicas(CVR) are scheduled on cStor pools **cspc-stripe-pool-6qkw** and **cspc-stripe-pool-pn9p** from the **status.poolInfo** since above pools are on **e2e1-node1** and **e2e1-node2** (able to find node info from cspi output) sodata also will be available on the same node. Info under spec i.e **spec.policy.replicaPoolInfo** will convey where to schedule cStorVolumeReplicas.
```